Description: Fixes the wrong revision number in the build file
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
Origin: vendor
--- a/build.xml
+++ b/build.xml
@@ -55,7 +55,7 @@
   <property name="component.title"         value="Commons IO"/>
 
   <!-- The current version number of this component -->
-  <property name="component.version"       value="2.2-SNAPSHOT"/>
+  <property name="component.version"       value="2.4"/>
 
   <!-- The base directory for component sources -->
   <property name="source.home"             value="src/main/java"/>
